lib/backend/model/statusreports.go

```diff
@@ -59,10 +60,11 @@ func (key ActiveStatusReportKey) String() string {
 
 type ActiveStatusReportListOptions struct {
 	Hostname string
+	Region   string
 }
 
 func (options ActiveStatusReportListOptions) defaultPathRoot() string {
-	k := "/calico/felix/v1/host"
+	k := "/calico/felix/v2/" + regionString(options.Region) + "/host"
```

Comment:
As discussed, code does what felix needs but it's breaking the normal convention in this API (where empty string means "don't filter on this field").

Reply:
Fixed in next commit by changing the `Region` fields to `RegionString`.  A region _string_ is never validly empty.